api/Headers.json

```diff
@@ -0,0 +1,1106 @@
+{
+  "api": {
+    "Headers": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Headers",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
+          },
+          "chrome": {
+            "version_added": "42"
+          },
+          "chrome_android": {
+            "version_added": "42"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": [
+            {
+              "version_added": "52",
+              "notes": "Prior to Firefox 52, <code>get()</code> only returned the first value in the specified header, with <code>getAll()</code> returning all values. From 52 onwards, <code>get()</code> now returns all values and <code>getAll()</code> has been deleted."
+            },
+            {
+              "version_added": "39"
+            },
+            {
+              "version_added": "34",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": ""
+                }
+              ]
+            }
+          ],
+          "firefox_android": {
+            "version_added": "44"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "29"
+          },
+          "opera_android": {
+            "version_added": "29"
+          },
+          "safari": {
+            "version_added": "10.1"
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "advanced_header_methods": {
```

Comment:
Remove the whole "advanced_header_methods entry. It is redundant with the `entries`, `keys`, `values` data.

Reply:
👍